Extension/package.json

```diff
@@ -1762,7 +1762,7 @@
     "mkdirp": "^0.5.1",
     "plist": "^3.0.1",
     "tmp": "^0.1.0",
-    "vscode-cpptools": "^3.0.1",
+    "vscode-cpptools": "^3.1.0",
```

Comment:
Shouldn't this cause an update to yarn.lock also?

FYI, the API change was a breaking change.  The "^" implies "use at least this version, up until the next major version number".  So, anyone already using "^3.x.x" (which is probably the common approach) will receive the new version.  Can we withdraw 3.1.0 and republish as 4.0?

Reply:
Since there is a new event in [CppToolsTestHook](https://github.com/microsoft/vscode-cpptools-api/blob/ca992e4637cf5f0e576d9bb721fea792e97935d4/testApi.ts#L37), any class that implements it and depends on "^3.x.x" will need to use the new event as well. 

If the c/c++ extension is the only extension that's using `CppToolsTestHook`, then I think it's okay to keep the API as 3.1.0 and update the c/c++ extension's dependency to "^3.1.0". However, to not break those using the CppToolsTestHook API, then the new event should be published under 4.0. 

@bobbrow, thoughts?

Reply:
I'll update yarn.lock when we decide on what version of API to publish and use.

Reply:
Discussed offline. Only c/c++ extension is using `CppToolsTestHook`, so only need to update dependency to "^3.1.0".